database/migrations/2023_06_19_113411_social_logins.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('social_logins', function (Blueprint $table) {
+            $table->increments('id');
```

Comment:
By default id will always incremental. So use unsignedBigInteger(). Call nullable() in client_id line too